npm module for 'card' does not load via require('card')

Hi there - thanks for card! I was happy to see an NPM module for card, I'm using browserify in a project and having card available on npm (along with jquery, ractive, and everything else I'm using) is very handy. However the 'card' module seems broken - it simply doesn't load via the normal `require('card')`:

```
$ npm update card
npm http GET https://registry.npmjs.org/card
npm http 304 https://registry.npmjs.org/card

$ ls node_modules/card/
LICENSE         bower.json      card.sublime-workspace  gulpfile.coffee     node_modules        src
README.md       card.sublime-project    example         lib         package.json

$ node

> require('card')
Error: Cannot find module 'card'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at repl:1:2
    at REPLServer.self.eval (repl.js:110:21)
    at Interface.<anonymous> (repl.js:239:12)
    at Interface.emit (events.js:95:17)
    at Interface._onLine (readline.js:202:10)
    at Interface._line (readline.js:531:8)
```

Other modules load fine. This bug should be pretty easy to replicate. I'm investigating further to find out the cause of the prob.

--- a/package.json
+++ b/package.json
@@ -2,6 +2,7 @@
   "name": "card",
   "version": "0.1.3",
   "author": "Jesse Pollak <jpollak92@gmail.com>",
+  "main": "lib/js/card.js",
   "description": "Card let's you add an interactive, CSS3 credit card animation to your credit card form to help your users through the process.",
   "contributors": [
     {

